src/fragments/lib/auth/ios/signin/50_multi_factor_confirm_signin.mdx

```diff
@@ -1,16 +1,14 @@
 <BlockSwitcher>
 
-<Block name="Listener (iOS 11+)">
+<Block name="Async/Await (iOS 13+)">
```

Comment:
Do we need to give 13+ here, because Amplify iOS will be available on in 13.0, we do not need to explicitly add it here.

Reply:
I've removed "(iOS 13+)" from both the the async/wait and Combine block names